### PR TITLE
Convert all presets to TypeScript

### PR DESCRIPTION
**Pull request by mxstbr**

### CODE REVIEW

**Comment by mxstbr** [comment]
These build failure logs are especially unhelpful 🤔 

```
$ microbundle --no-compress
Build "typography" to dist:
       2.2 kB: index.js.gz
       1.9 kB: index.js.br
      2.13 kB: index.esm.js.gz
      1.84 kB: index.esm.js.br
      2.36 kB: index.umd.js.gz
      2.05 kB: index.umd.js.br
Done in 4.88s.

Received "killed" signal
```

From [these support posts on the CircleCI forum](https://discuss.circleci.com/t/kill-signal-problem/18665/6) it sounds like that means the process is running out of memory?! How's that possible while building a handful of packages?

File: packages/preset-dark/src/index.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
       my: 4,
       borderCollapse: 'separate',
       borderSpacing: 0,
-      [['th', 'td']]: {
+      [['th', 'td'].toString()]: {
```
Comment (jxnblk) [inline packages/preset-dark/src/index.ts]: Did you decide not to go this route here?
```suggestion
      'th, td': {
```

**Comment by jxnblk** [comment]
:+1: on the actions setup. We can cleanup the circle ci config after the currently open PRs are merged 

Overall LGTM :shipit:

File: packages/preset-dark/src/index.ts (hunk 1 above)
Comment (mxstbr) [reply packages/preset-dark/src/index.ts]: Okay boss I did it properly

**Pull request merged**